reference/create-reports.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "Create Evaluation Report"
+api: "POST https://api.promptlayer.com/reports"
+---
+
+This endpoint allows for the creation of Evaluation reports. These reports are given a name and associated with a dataset.
```

Comment:
This endpoint allows for the creation of Evaluation pipelines. These pipelines are given a name and associated with a dataset.